bigbluebutton-html5/private/config/settings.yml

```diff
@@ -973,6 +973,13 @@ public:
       - coffeeshop.jpg
       - board.jpg
 private:
+  postgresql:
+    host: localhost
+    port: 5432
+    database: bbb_graphql
+    user: bbb_frontend
+    password: bbb_frontend
+    timeout: 30000
```

Comment:
I think we should stop using `settings.yml` to store server configs. Once it is shared with all users thought Graphql!

Reply:
Alternatively we can remove configs below `private:` from being shared through Graphql!